packages/legend-query-builder/src/components/fetch-structure/QueryBuilderPostFilterPanel.tsx

```diff
@@ -802,6 +822,9 @@ const QueryBuilderPostFilterPanelContent = observer(
       }
     };
     const newGroupWithCondition = (): void => {
+      QueryBuilderTelemetry.logEvent_PostFilterCreateGroupFromConditionLaunched(
+        applicationStore.telemetryService,
```

Comment:
Not sure if the metrics around all the options in filter panels is that useful to draw some meaningful analysis

Reply:
Yea, the only reason we add the metrics right now I believe is to determine which if any merits their own icon since we're consolidating all the previously displayed 7 icon options into a single '...' option 🤔.